src/appleseed.studio/utility/inputwidgetproxies.cpp

```diff
@@ -308,7 +308,7 @@ namespace
                     output_spectrum);
 
                 Color3f ciexyz;
-                spectrum_to_ciexyz_standard(&output_spectrum[0], &ciexyz[0]);
+                spectral_reflectance_to_ciexyz_standard(&output_spectrum[0], &ciexyz[0]);
```

Comment:
Add a `// todo:` because this isn't correct when picking a color for an emitter.